charts/testkube/values.yaml

```diff
@@ -355,6 +355,24 @@ testkube-api:
     digest: ""
     # -- Testkube API k8s secret for private registries
     pullSecrets: []
+
+  # Test Workflows image parameters
+  imageWorkflows:
```

Comment:
I think that these should be separate:
1. To allow a specific tag for each
2. To keep common syntax
3. To make `digest` working correctly, as they cannot share the same digest

Reply:
Done